lom/src/lib/lomcommon/helper.go

```diff
@@ -493,6 +494,43 @@ func UpdatePeriodicLogTime(id string, newPeriod int) error {
     return GetlogPeriodic().updatePeriod(id, newPeriod)
 }
 
+/* Helper to add a periodic log entry with short timeout and long timeout
+* This function will add periodic log entry with short timeout and then update the timeout to long timeout
+* after short timeout expiry. This function will also listen for stop signal to remove the periodic log entry
+* before long timeout expiry.
+* Note : This is one time call API. Usechannel to stop the logperiodic. Do not remove periodic entry or update timeout
+* Min time >= MIN_PERIODIC_LOG_PERIOD_SECS
+ */
+func AddPeriodicLogWithTimeouts(ID string, message string, shortTimeout time.Duration,
+    longTimeout time.Duration) chan bool {
+    // Create a channel to listen for stop signals to kill timer
+    stopchannel := make(chan bool)
+
+    GetGoroutineTracker().Start("AddPeriodicLogWithTimeouts"+ID+GetUUID(), func() {
```

Comment:
can make this string literal.

Reply:
May be it will become verbose. Since the whole purpose here is to get unique ID. SO if we create an function which return string, again we need to pass API name, additional arguments. It will still become verbose. I will try to add better helper API in goroutine tracker later